kohese-portal/client/src/components/import/import.component.html

```diff
@@ -18,6 +18,7 @@
 <div [ngStyle]="{ 'height': isDialogInstance() ? '40%' : '45%' }" style="width: 100%;">
   <h3 style="height: 10%;">Parent</h3>
   <tree #itemProxyTree style="height: 87%;"
+    [isImport]="!!true"
```

Comment:
What is the purpose of !!true, wouldn't that be true?

Reply:
It appears to be required to ensure that the return is a boolean true.

Reply:
it wasn't being sent correctly when i just sent 'true'. so saying 'not, not true' made sure to type it as boolean and true.